.circleci/config.yml

```diff
@@ -33,13 +33,13 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - v2-3scaleoperator-vendor-{{ arch }}-{{ checksum "go.sum" }}
+            - v3-3scaleoperator-vendor-{{ arch }}-{{ checksum "go.sum" }}
```

Comment:
why?

Reply:
We are caching vendor directory, however we are now using more packages in a "module"(or repo) that we were not using before. Due to the vendor directory was cached the new files belonging to this repository were not used.
It is the case of '  github.com/openshift/client-go v3.9.0+incompatible ' in go.mod. We still need to use this, but now we use an additional package there.
By increasing the version number we force CircleCI to redownload the dependencies and are now included.